README.md

```diff
@@ -0,0 +1,72 @@
+# Budget App
+
+> An application built using Ruby on Rails to manage the expenses of any user. It allows the user the create categories under which are various transactions for the category. 
+
+## Screenshot
+| Categories Page |   Transactions page |
+| ---------- | ----------- |
+ | ![CategoriesPage](img/1.png) | ![TransactionsPage](img/2.png) |
+</p>
+
+
+## Built With
+
+- Ruby on Rails
+- Ruby
+- Rubocop
+
+## Getting Started
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+``VS Code or other Code Editor``
+
+### Setup
+
+``Install Postgres and Ruby applications``
+
+### Install
+
+    git clone https://github.com/Theophileaseh/Budget-app.git
+
+### Usage
+
+``cd Budget-app``
+
+``bundle install``
+
+``Run "rails server or rails s" to start the application on local server``
+
+### Run tests
+
+``Run "rspec" in your terminal``
+
+
+## Authors
+
+👤 **Nwachan Theophile**
+
+- GitHub: [@Theophileash](https://github.com/Theophileaseh)
+- Twitter: [@NwachanT](https://twitter.com/NwachanT)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/nwachan-theophile)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
```

Comment:
- [ ] As it is  a [requirement](https://gitlab.com/microverse/guides/tse/code_review/code_review_guidelines_2_0/-/blob/master/ror/capstone/ror_capstone.md) for this project, " `The Creative Commons license of the design requires that you give appropriate credit to the author. Therefore, you must do it in the README of your project.` ", kindly give appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).

Reply:
As requested by the previous reviewer, kindly give the credit to the author of the design in the Readme file. Kindly do so and then submit for a re-review.